### PR TITLE
Adding Requirements.txt & update .gitignore

### PR DESCRIPTION
**Pull request by kylekap**

### CODE REVIEW

**Comment by kylekap** [comment]
Created virtual environment, installed necessary imports, created requirements.txt file.

updated .gitignore to include standard python exclusions (due to virtual environment)

updated readme to remove line about requirements.txt file.

File: scraper/Requirements.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+certifi==2021.10.8
```
Comment (ProstoSanja) [inline scraper/Requirements.txt:1]: A lot of the stuff here has nothing to do with the project, certifi? Did you just run pip freeze on your entire system? 

This project needs like max websockets and that's about it.

File: .gitignore [hunk 2]
```diff
@@ -25,4 +25,119 @@ yarn-error.log*
...
+coverage.xml
+*.cover
+.hypothesis/
+.pytest_cache/
+
+# Translations
+*.mo
+*.pot
+
+# Django stuff:
```
Comment (ProstoSanja) [inline .gitignore:84]: Yeah, can we just keep parts that are relevant? We're not using Django anywhere here and won't be. So no need for a massive gitignore, if we just want to deal with requirements and venvs

**Review by ProstoSanja — CHANGES_REQUESTED** [review]

File: scraper/Requirements.txt (hunk 1 above)
Comment (kylekap) [reply scraper/Requirements.txt:1]: You've got requests which by default includes a lot of those other things. Same for websocket, has a few dependencies

Did you mean to use websocket-client? That has fewer dependencies, but is not what was utilized here.

Comment (kylekap) [reply scraper/Requirements.txt:1]: [requirements justifications.txt](https://github.com/ProstoSanja/place-2022/files/8417655/requirements.justifications.txt)

Comment (ProstoSanja) [reply scraper/Requirements.txt:1]: Ah, i apologise, im not really familiar with pythion, venvs, so thought it only needed a list of direct dependencies, and not entire tree. Just checked locally, all good.

**Review by ProstoSanja — APPROVED** [review]

**Pull request merged**